tools/csv-merger/csv-merger.go

```diff
@@ -623,6 +623,7 @@ The NooBaa operator deploys and manages the [NooBaa][2] Multi-Cloud Gateway on O
 	// This can be used to enable/disable console support for any supported feature
 	// Example: "features.ocs.openshift.io/enabled": `["external", "foo1", "foo2", ...]`
 	ocsCSV.Annotations["features.ocs.openshift.io/enabled"] = `["external"]`
+	ocsCSV.Annotations["external.ocs.openshift.io/validation"] = `{"encodedKeys":["rook-ceph-operator-creds", "rook-csi-rbd-node", "rook-csi-rbd-provisioner", "rook-csi-cephfs-node", rook-csi-cephfs-provisioner"], "plainKeys": ["rook-ceph-mon-endpoints", "rook-ceph-retain-bucket", "rook-ceph-mon"]}`
```

Comment:
I would rather just have a map of `kind` to a list of names.
```suggestion
	ocsCSV.Annotations["external.ocs.openshift.io/validation"] = `{"secrets":["rook-ceph-operator-creds", "rook-csi-rbd-node", "rook-csi-rbd-provisioner", "rook-csi-cephfs-node", rook-csi-cephfs-provisioner"], "configMaps": ["rook-ceph-mon-endpoints", "rook-ceph-mon"], "storageClasses":["rook-ceph-retain-bucket"]}`
```
I would also change the annotation key to `external.features.ocs.openshift.io/jsonValidation`.

In addition, this needs a comment in the code explaining what it does and who uses it.